frontend/gateway/gateway.go

```diff
@@ -640,6 +648,40 @@ func (lbf *llbBridgeForwarder) Solve(ctx context.Context, req *pb.SolveRequest)
 	var defaultID string
 
 	lbf.mu.Lock()
+
+	if res.Attestations != nil {
+		attests := map[string]*pb.Attestations{}
```

Comment:
I see this as a verb and find confusing. Better use full noun, it's not that much longer to type.